transform/mattermost-analytics/models/marts/data_eng/event_registry/dim_events.sql

```diff
@@ -0,0 +1,16 @@
+{{
+    config({
+        "tags":"hourly",
+        "cluster_by": ['event_name', 'source'],
```

Comment:
are there any pros/cons to cluster by `event_name` vs. `event_id`?

Reply:
Event id is unique and a string. I'm not sure that it offers good granularity, as it doesn't really allow to group events in the dimension. 

TBH, given the current number of rows in this table, I'm not sure it will make an important difference. We can drop clustering for now.

Reply:
Good point, wondering what clustering really means for dimension tables, or is it just a "smth. one can do" kind of thing.

Reply:
I will remove it for now and we can reconsider if we see performance issues.

Reply:
Removed the clustering and fixed a bug in grouping.